helm-chart/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: Nextcloud AIO Helm Chart
 description: A generated Helm Chart for Nextcloud AIO from Skippbox Kompose
-version: 4.0.1
+version: 4.2.0
 apiVersion: v1
```

Comment:
This could be v2 https://helm.sh/docs/topics/charts/#the-apiversion-field

Reply:
Are you completely sure? Then I can add this. I just used the files that [kompose](https://github.com/kubernetes/kompose) generated automatically.

Reply:
Correct me if I'm wrong: I think this file is not generated by kompose as kompose is not aware of helm charts.

Reply:
It actually is able to do so :)
https://github.com/nextcloud/all-in-one/blob/6ece0126809374350239c9811b96dc7dbf6cd1c5/helm-chart/update-helm.sh#L37

Reply:
![image](https://user-images.githubusercontent.com/42591237/215606374-a5077973-b304-4fdc-93bc-21da30ccd4b9.png)

Reply:
Is this fine @mrueg ?
https://github.com/nextcloud/all-in-one/pull/1848/commits/c8576044b23dd2bd5b5fb80f02b96e4c90fa49a4